sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PortEvaluationStage.java

```diff
@@ -34,48 +31,70 @@
     private final String portName;
     private final int port;
     private final Optional<String> customEnvKey;
+    private final boolean useHostPorts;
 
     private String resourceId;
 
     public PortEvaluationStage(
-            Protos.Resource resource, String taskName, String portName, int port, Optional<String> customEnvKey) {
+            Protos.Resource resource,
+            String taskName,
+            String portName,
+            int port,
+            Optional<String> customEnvKey,
+            boolean useHostPorts){
```

Comment:
Hyper-nit: space before `{` (I'll make a note to get that into PMD or checkstyle if possible)